conans/client/conf/config_installer.py

```diff
@@ -207,6 +211,34 @@ def _save_configs(configs_file, configs):
                                   indent=True))
 
 
+def _get_current_time_now():
+    return datetime.now(gettz())
+
+
+def _next_scheduled_config_install(sched_path, value, interval):
+    """ Return when the next config install should occur.
+        In case of error, the next config install should occur immediately
+
+    :param sched_path: sched file path in conan cache
+    :param value: time interval value e.g 5 from 5m
+    :param interval: time interval unit e.g m from 5m
+    :return: datetime instance + time interval
+    """
+    timestamp = os.path.getmtime(sched_path)
+    sched = datetime.fromtimestamp(timestamp, tz=gettz())
+    if interval == 'm':
+        sched += timedelta(minutes=float(value))
+    elif interval == 'h':
+        sched += timedelta(hours=float(value))
+    else:
+        sched += timedelta(days=float(value))
+    return sched
+
+
+def _get_config_install_interval(config):
```

Comment:
This should be better implemented in the config class and return None if not defined.